src/pages/team/single-team.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
This page naming is wrong. I thought you how to name a dynamic page.

use `[id].tsx`

Reply:
I am yet to understand it's pros and cons sir. Even had issues generating a received name property from the team card. Instead it was displaying the ```name``` as ```team-profile``` since I set the href as ```team/team-profile```, and named the dynamic page as [name].jsx

Why use a dynamic page instead of a normal component and when should I always fall back to this approach?